app/views/providers/merits_task_lists/show.html.erb

```diff
@@ -13,17 +13,41 @@
         <%= @merits_task_list.errors[:task_list].first %>
       </p>
     <% end %>
-    <ol class="app-task-list">
-      <%= render "task_list_section", name: t(".application_section_heading"), ccms_code: nil, tasks: @merits_tasks[:application], section_number: 1 %>
-      <li>
-        <h2 class="govuk-heading-l govuk-!-margin-top-4"><%= t(".proceeding_heading") %></h2>
-      </li>
-      <% @merits_tasks[:proceedings].each_with_index do |(_, proceeding), index| %>
-        <%= render "task_list_section", name: proceeding[:name], tasks: proceeding[:tasks], ccms_code: @merits_tasks[:proceedings].keys[index], section_number: index + 2 %>
-      <% end %>
-    </ol>
+
+    <h3 class="govuk-heading-m">1. <%= t(".application_section_heading") %></h3>
+
+    <%= govuk_task_list(id_prefix: "task-list_case-details") do |task_list|
+          @merits_tasks[:application].each do |task|
+            task_list.with_item(
+              title: t(".#{task.name}"),
+              href: _task_url(task.name, @legal_aid_application, task.state),
+              status: task.state == :complete ? t(".states.#{task.state}") : govuk_tag(text: t(".states.#{task.state}")),
+            )
+          end
+        end %>
+
+    <h2 class="govuk-heading-l"><%= t(".proceeding_heading") %></h2>
+    <% @merits_tasks[:proceedings].each_with_index do |(ccms_code, proceeding), index| %>
+      <h3 class="govuk-heading-m"><%= index + 2 %>. <%= proceeding[:name] %></h3>
```

Comment:
`<%= index + 2 %>`  count seems off for heading sections, should count start from `1`?

Reply:
There's already a sneaky "1" on line 17. If this isn't immediately obvious, have you got any suggestions on how to make it clearer?

Reply:
Thinking about this more, it should probably be a `<ol>`, but I don't know a way of doing this because the headers (e.g. "Enter general case details") would sit outside the list. Maybe the page should be redesigned to accommodate this. What do you think @naseberry

Reply:
I could do something like `<ol start="2">` but that doesn't feel right

Reply:
Good thought process, lets discuss.

My original comment was more about the logical order of number.
Why would the section start from any number but 1, it seemed odd.